share/man/man7/arch.7

```diff
@@ -192,28 +216,30 @@ Most ILP32 ABIs, except
 require only 4-byte alignment for 64-bit integers.
 .Pp
 Machine-dependent type sizes:
-.Bl -column -offset indent "Architecture" "void *" "long double" "time_t"
-.It Sy Architecture Ta Sy void * Ta Sy long double Ta Sy time_t
-.It aarch64     Ta 8 Ta 16 Ta 8
-.It amd64       Ta 8 Ta 16 Ta 8
-.It armv6       Ta 4 Ta  8 Ta 8
-.It armv7       Ta 4 Ta  8 Ta 8
-.It i386        Ta 4 Ta 12 Ta 4
-.It mips        Ta 4 Ta  8 Ta 8
-.It mipsel      Ta 4 Ta  8 Ta 8
-.It mipselhf    Ta 4 Ta  8 Ta 8
-.It mipshf      Ta 4 Ta  8 Ta 8
-.It mipsn32     Ta 4 Ta  8 Ta 8
-.It mips64      Ta 8 Ta  8 Ta 8
-.It mips64el    Ta 8 Ta  8 Ta 8
-.It mips64elhf  Ta 8 Ta  8 Ta 8
-.It mips64hf    Ta 8 Ta  8 Ta 8
-.It powerpc     Ta 4 Ta  8 Ta 8
-.It powerpcspe  Ta 4 Ta  8 Ta 8
-.It powerpc64   Ta 8 Ta  8 Ta 8
-.It powerpc64le Ta 8 Ta  8 Ta 8
-.It riscv64     Ta 8 Ta 16 Ta 8
-.It riscv64sf   Ta 8 Ta 16 Ta 8
+.Bl -column -offset indent "Architecture" "long" "void *" "long double" "time_t"
```

Comment:
Most of this hunk is adding a column for `long` as it is now more arch-dependent.